src/v2/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -272,7 +272,7 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
               )}
               {newOfferSubmissionEnabled && !isInquiryCheckout && !isEdition && (
                 <>
-                  <Text variant="lg" color="black80" marginTop={4}>
+                  <Text variant="lg" color="black80" marginTop={2}>
```

Comment:
We should change the `marginTop` to `mt` if that's the correct one to use, as in line `154`